homeassistant/components/tesla/config_flow.py

```diff
@@ -0,0 +1,160 @@
+"""Tesla Config Flow."""
+import logging
+
+import voluptuous as vol
+
```

Comment:
Please remove this blank line.

I recommend the isort tool to automatically sort imports.